packages/app/src/pages/internationaal/positief-geteste-mensen.tsx

```diff
@@ -137,3 +212,77 @@ export default function PositiefGetesteMensenPage(
     </Layout>
   );
 }
+
+function compileInternationalData(data: Record<string, In>) {
+  const flattenedData: CompiledCountriesValue[] = [];
+
+  /**
+   * @todo
+   * For other pages, this method can accept the metricName and metricProperty
+   * as arguments; combined with proper typing.
+   */
+  const metricName = 'tested_overall';
+  const metricProperty = 'infected_per_100k_average';
+
+  Object.entries(data).forEach(([countryCode, countryData]) => {
+    countryData[metricName].values.forEach((value: InTestedOverallValue) => {
+      let objectMatch = flattenedData.find(
+        (o) =>
+          o.date_start_unix === value.date_start_unix &&
+          o.date_end_unix === value.date_end_unix
+      );
+
+      if (!isDefined(objectMatch)) {
+        objectMatch = {
+          date_start_unix: value.date_start_unix,
+          date_end_unix: value.date_end_unix,
+        } as CompiledCountriesValue;
+
+        flattenedData.push(objectMatch);
+      }
+
+      objectMatch[countryCode as CountryCode] = value[metricProperty];
+    });
+  });
+
+  const sortedData = flattenedData.sort((a, b) => {
+    return a.date_start_unix - b.date_start_unix;
+  });
+
+  return sortedData;
+}
+
+function selectedCountriesToSeriesConfig(
+  selectedCountries: CountryCode[],
+  countryNames: Record<CountryCode, string>,
+  selectionColors: string[]
+): LineSeriesDefinition<CompiledCountriesValue>[] {
+  return [
+    {
+      type: 'line' as const,
+      metricProperty: 'nld' as CountryCode,
+      label: countryNames['nld'],
+      color: colors.data.neutral,
+    } as LineSeriesDefinition<CompiledCountriesValue>,
+  ].concat(
+    selectedCountries.map((countryCode, index) => ({
+      type: 'line' as const,
+      metricProperty: countryCode,
+      label: countryNames[countryCode],
+      color: selectionColors[index],
+    }))
+  );
+}
+
+function compileCountrySelectionData(
+  countryCodes: CountryCode[],
+  countryNames: Record<CountryCode, string>,
+  data: CompiledCountriesValue[]
+): Country[] {
+  const lastValues = last(data);
+  return countryCodes.map((countryCode) => ({
+    code: countryCode,
+    name: countryNames[countryCode],
+    lastValue: lastValues?.[countryCode] ?? 0,
+  }));
```

Comment:
I think an assert is more appropriate here than a fallback to 0

```suggestion
  const lastValues = last(data);
  assert(lastValues, 'Data appears to be empty')
  
  return countryCodes.map((countryCode) => ({
    code: countryCode,
    name: countryNames[countryCode],
    lastValue: lastValues[countryCode],
  }));
```